CyberEyes/CyberEyes/CustomizationPage.xaml.cs

```diff
@@ -16,5 +16,50 @@ void Handle_StartClicked(object sender, System.EventArgs e)
 		{
 			Navigation.PushAsync(new GamePage(), true);
 		}
+
+		void Handle_EasyTapped(object sender, System.EventArgs e)
+		{
+			easyFrame.BackgroundColor = Color.Accent;
+			var label = easyFrame.Content as Label;
+			label.Text = "● Easy";
```

Comment:
That's a good idea. I was unsure if we needed to do it for this milestone but it won't be difficult to add. I'll take care of that next